metrics/stackdriver_exporter.go

```diff
@@ -102,11 +102,12 @@ func init() {
 
 func newOpencensusSDExporter(o stackdriver.Options) (view.Exporter, error) {
 	e, err := stackdriver.NewExporter(o)
-	if err == nil {
-		// Start the exporter.
-		// TODO(https://github.com/knative/pkg/issues/866): Move this to an interface.
-		e.StartMetricsExporter()
+	if err != nil {
+		return nil, err
 	}
+	// Start the exporter.
+	// TODO(https://github.com/knative/pkg/issues/866): Move this to an interface.
+	e.StartMetricsExporter()
```

Comment:
Well then we should consume the error from `StartMetricsExporter` as well.

Reply:
Good catch, I didn't realize it returned an `error`.